modules/http4s/src/smithy4s/http4s/internals/SmithyHttp4sClientEndpoint.scala

```diff
@@ -35,6 +35,7 @@ import smithy4s.schema.SchemaAlt
   */
 // format: off
 private[smithy4s] trait SmithyHttp4sClientEndpoint[F[_], Op[_, _, _, _, _], I, E, O, SI, SO] {
+  def build(input: I): (Request[F], Response[F] => F[O])
```

Comment:
I guess this is BC as we're in a `private[smithy`] trait

Reply:
Well adding methods doesn't break backward compat, that's for certain

Reply:
abstract ones? In general I believe it's breaking (code compiled against the previous version would have a missing method impl), here you could consider it's safe cause the trait is package-private